app/uk/gov/hmrc/cgtpropertydisposalsfrontend/config/ViewConfig.scala

```diff
@@ -185,4 +185,9 @@ class ViewConfig @Inject() (servicesConfig: ServicesConfig) extends Logging {
   val selfAssessmentTaxReturnsCorrections: String = getString("external-url.sa-tax-returns-corrections")
 
   val numberOfTaxYearsForReturns: Long = servicesConfig.getInt("returns-number-of-tax-years").toLong
+
+  val draftReturnNewDueDateStartYear: Int  = servicesConfig.getInt("draft-return.new-due-date.start-year")
```

Comment:
For myself, configuring a date, I would configure a String and parse that, rather than 3 integers

Reply:
CGT has lots of configurations in this format.